Dockerfile

```diff
@@ -34,6 +34,12 @@ FROM ruby:3.4.2-alpine3.21 as production
 # Install PostgreSQL client tools
 RUN apk add --no-cache postgresql-client
 
+# Install bash
```

Comment:
So rather than creating additional layers we can just add this to the list of installed packages in the existing apk add line